translations/messages.en.yml

```diff
@@ -683,6 +683,8 @@ flashes:
         notice:
             tag_added: Tag added
             tag_renamed: 'Tag renamed'
+            too_much_tags: To avoid perfomance issues, you can't add too much tags (more than %tags%) or too long tags (%characters% characters).
```

Comment:
I would maybe see a text like:

> To avoid performance issues, you can't add more than %tags% at once or tags having more than %characters%

Reply:
Thank you (and I have a typo in `performance`).